ambry-clustermap/src/main/java/com.github.ambry.clustermap/StaticClusterManager.java

```diff
@@ -533,6 +533,12 @@ public JSONObject getSnapshot() {
     return snapshot;
   }
 
+  @Override
+  public ReplicaId getNewReplica(String partitionIdStr, DataNodeId dataNodeId) {
+    throw new UnsupportedOperationException(
+        "Adding new replica is currently not supported in static cluster manager. Return null here");
```

Comment:
Should this return null like the exception message says?

Reply:
updated the exception message. Feel like `return null` is unneeded.